index.js

```diff
@@ -14,4 +14,150 @@ menuItem.forEach((menuItem) => {
+    liveLink: '',
+    sourceLink: '',
+  },
+  {
+    name: 'Multi-Post Stories',
+    description: 'Lorem Ipsum is simply dummy text of the prinhe 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent.',
+    image: './img/Dbg2.png',
+    stack: ['FACEBOOK', 'Full Stack Dev', '2015'],
+    technologies: ['HTML', 'CSS', 'JavaScript', 'Ruby', 'Bootstrap'],
+    liveLink: '',
+    sourceLink: '',
+  },
+  {
+    name: 'Facebook 360',
+    description: 'Lorem Ipsum is simply dummy text of the prinhe 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent.',
+    image: './img/Dbg3.png',
+    stack: ['FACEBOOK', 'Full Stack Dev', '2015'],
+    technologies: ['HTML', 'CSS', 'JavaScript', 'Ruby', 'Bootstrap'],
+    liveLink: '',
+    sourceLink: '',
+  },
+  {
+    name: 'UBER Navigation',
+    description: 'Lorem Ipsum is simply dummy text of the prinhe 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent.',
+    image: './img/Dbg4.png',
+    stack: ['FACEBOOK', 'Full Stack Dev', '2015'],
+    technologies: ['HTML', 'CSS', 'JavaScript', 'Ruby', 'Bootstrap'],
+    liveLink: '',
+    sourceLink: '',
+  },
+];
+
+const cardsContainer = document.querySelector('#About');
+
+projects.forEach((project) => {
+  const card = document.createElement('div');
+  card.classList.add('about-section');
+
+  card.innerHTML = `
+    <img src="${project.image}" class="card-img" />
+      <div class="canopy">
+        <h4 class="about-head">${project.name}</h4>
+        <ul class="cano">
+          <li class="tec">FACEBOOK</li>
+          <li class="tech">Full Stack Dev</li>
+          <li class="tech">2015</li>
+        </ul>
+        <p class="canopy-txt">
+        A daily selection of privately personalized reads; no accounts or
+          sign-ups required
+        </p>
+        <ul class="tech1">
+          <li class="tech-t">HTML</li>
+          <li class="tech-r">Ruby On Rails</li>
+          <li class="tech-t">CSS</li>
+          <li class="tech-j">JavaScript</li>
+        </ul>
+        
+        <button type="button" class="project-btn">See Project</button>
+      </div>
+  `;
+
+  cardsContainer.appendChild(card);
+
+  const button = card.querySelector('.project-btn');
+
+  const modalContainer = document.createElement('div');
+  modalContainer.classList.add('modal');
+  modalContainer.id = 'modal';
+
+  modalContainer.innerHTML = `
+  <div class="modal-content">
+    <span class="close">&times;</span>
+    <h2 class="project-title" id="projectTitle"></h2>
+    <ul id="cano">
+    </ul>
```

Comment:
- [ ] Well! This list is okay, but there are still some adjustments needed in the margin and font styles. 🙏 Please closely follow the design and update your code accordingly.

| Your | Professional |
| --- | --- |
| ![Group 68](https://github.com/Danny-imo/my-project/assets/55840999/20bfd692-5d5f-462e-9f43-4eedb0297ae0) | ![Group 65](https://github.com/Danny-imo/my-project/assets/55840999/141d2efa-b610-47a3-afe3-dda248591000) |

By making these adjustments, you will ensure that the list matches the design more closely in terms of margin and font styles. Good luck with your updates! ✨🎨